tests/test_rex.py

```diff
@@ -255,73 +255,73 @@ def test_arbitrary_transmit():
     """
     _do_arbitrary_transmit_test_for("tests/i386/arbitrary_transmit")
 
-def test_KPRCA_00057():
```

Comment:
If these are broken, the convention is not to comment them out but rather to just rename them to `broken_*` instead of `test_*`.

Reply:
Sure, I just forgot the convention. Sorry again....